modules/cells/src/main/java/dmg/cells/nucleus/SystemCell.java

```diff
@@ -356,15 +356,15 @@ public void uncaughtException(Thread t, Throwable e)
         Throwable root = Throwables.getRootCause(e);
         if (root instanceof FileNotFoundException) {
             if (root.getMessage().contains("Too many open files")) {
-                _log.error(AlarmMarkerFactory.getMarker(PredefinedAlarm.OUT_OF_FILE_DESCRIPTORS,
+                logger.error(AlarmMarkerFactory.getMarker(PredefinedAlarm.OUT_OF_FILE_DESCRIPTORS,
                                                         getCellDomainName(),
                                                         getCellName()),
-                           "Uncaught exception in thread " + t.getName(),
+                           "Uncaught exception in thread {}", t.getName(),
```

Comment:
We checked again this case and got the following which we think show that the FAQ from slf4j is right:

`package diskCacheV111.admin;

import org.slf4j.Logger;
import org.slf4j.LoggerFactory;

public class TestClass {

    private static final Logger _log =
            LoggerFactory.getLogger(TestClass.class);
    public static void main(String [] args)
    {
        String name = "NAME";
        String name2 = "NAME2";

        _log.error("InterruptedException while waiting for a reply from pools " + new Exception("TESTTESTETST"));

        _log.error("InterruptedException while waiting for a reply from pools", new Exception("TESTTESTETST"));

        _log.error("InterruptedException while waiting for a reply from pools {}", new Exception("TESTTESTETST").toString());

        _log.error("InterruptedException while waiting for a reply from pools {}", name ,new Exception("TESTTESTETST"));

        _log.error("InterruptedException while waiting for a reply from pools {}, {}", name , name2, new Exception("TESTTESTETST"));
    }
}
`
The output:

`14:30:58.828 [main] ERROR diskCacheV111.admin.TestClass - InterruptedException while waiting for a reply from pools java.lang.Exception: TESTTESTETST
14:30:58.846 [main] ERROR diskCacheV111.admin.TestClass - InterruptedException while waiting for a reply from pools
java.lang.Exception: TESTTESTETST
	at diskCacheV111.admin.TestClass.main(TestClass.java:17) [classes/:na]
14:30:58.847 [main] ERROR diskCacheV111.admin.TestClass - InterruptedException while waiting for a reply from pools java.lang.Exception: TESTTESTETST
14:30:58.847 [main] ERROR diskCacheV111.admin.TestClass - InterruptedException while waiting for a reply from pools NAME
java.lang.Exception: TESTTESTETST
	at diskCacheV111.admin.TestClass.main(TestClass.java:21) [classes/:na]
14:30:58.847 [main] ERROR diskCacheV111.admin.TestClass - InterruptedException while waiting for a reply from pools NAME, NAME2
java.lang.Exception: TESTTESTETST
	at diskCacheV111.admin.TestClass.main(TestClass.java:23) [classes/:na]
`